### DISCUSSION TITLE
deployer v7 & symfony dependencies

### DISCUSSION THREAD

**Discussion by jonnitto** [deployphp/deployer#2342] [Help needed]
While testing the beta of version 7, I had some issues with the dependencies: Because of the release cycle, some tools/cms/packages still stick to `"symfony/console": ^4.2` and `"symfony/yaml": ^4.2`

It is really required to use version 5? Or would it be possible to set this to `^4.2 || ^5.0`?

**Comment by antonmedv** [comment]
Yes, I tried to go 4 || 5 way, but this is error prone and complicated. So now I specify exactly what Deployer is using. If you encounter install problems consider installing deployer/dist instead.

**Comment by jonnitto** [comment]
Ok, thank you for your fast answer.
